code/datums/components/storage/storage.dm

```diff
@@ -233,7 +233,7 @@
 
 /datum/component/storage/proc/quick_empty(mob/M)
 	var/atom/A = parent
-	if((!ishuman(M) && (A.loc != M)) || (M.stat != CONSCIOUS) || M.incapacitated())
+	if(M.canUseStorage() && (A.loc != M) && A.Adjacent(M) && M.incapacitated())

```

Comment:
Uh, I think you meant !canUseStorage and !Adjacent right ?

Reply:
h*ck, yea i got confused w that return below this line